crates/binstalk/src/drivers/registry.rs

```diff
@@ -56,6 +57,12 @@ pub enum RegistryError {
 
     #[error("Failed to render dl config: {0}")]
     RenderDlConfig(#[from] RenderError),
+
+    #[error("Failed to parse base16 checksum: {0}")]
+    InvalidBase16(#[from] Base16DecodeError),
```

Comment:
While technically correct, and matching the crate name, probably more friendly to use "hex" instead of "base16"

Reply:
Thanks, I've changed variant name and the error message.